Games/types/Mafia/roles/cards/Astral.js

```diff
@@ -0,0 +1,29 @@
+const Card = require("../../Card");
+const { PRIORITY_MODIFY_ACTION_LABELS } = require("../../const/Priority");
+
+module.exports = class Astral extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.actions = [
+            {
+                priority: PRIORITY_MODIFY_ACTION_LABELS,
+                labels: ["absolute", "hidden"],
+                run: function () {
+                    if (this.game.getStateName() != "Night")
+                        return;
+
+                    for (let action of this.game.actions[0]) {
+                        if (action.priority > this.priority &&
+                            action.actors.includes(this.actor)) {
+                            action.labels.push("hidden");
+                        }

```

Comment:
See if it is possible to make the action name have a [hidden] prefix 

cc @milkshakesatdawn (just keeping u in the loop, this change is not busywork)

Reply:
Would that just be a meeting mod then? And then use a method similar to the seance name change?

Reply:
I think you can use the existing card, listen for the rolesAssigned

Iterate through this.role.meetings and append the name like jailer and medium.

But I'm not sure if it will work. It's not an important fix, you can file it as a separate issue.